apps/api_web/lib/api_web/views/trip_view.ex

```diff
@@ -109,13 +109,17 @@ defmodule ApiWeb.TripView do
 
   @spec occupancies(Model.Trip.t(), Plug.Conn.t()) :: [Model.CommuterRailOccupancy.t()] | nil
```

Comment:
question: can this still return `nil`?

Reply:
Ah, I changed it from `nil` to `[]` to align with the type of the relationship, but forgot to change the spec!